src/components/demo/DOITooltipDemo.tsx

```diff
@@ -37,20 +37,68 @@ const DOITooltipDemo = (props: Props) => {
         }
     }
 
+    /* Function to map Handle or DOI response to valid record */
+    const FormatResponse = (responseRecord: {[name: string]: any}) => {
+        const record = {
+            data: {
+                attributes: {
+                    referentName: responseRecord.values[16].data.value,
+                    specimenHost: responseRecord.values[18].data.value,
+                    specimenHostName: responseRecord.values[19].data.value,
+                    primarySpecimenObjectId: responseRecord.values[20].data.value,
+                    topicDiscipline: responseRecord.values[27].data.value,
+                    livingOrPreserved: responseRecord.values[29].data.value
+                }
+            }
+        };
+
+        return record;
+    }
+
     /* Function for fetching DOI details */
     const TriggerTooltip = async () => {
         try {
-            const response = await fetch(`https://dev.dissco.tech/handle-manager/api/v1/pids/${doi.replace(process.env.REACT_APP_DOI_URL as string, '')}`);
-            const record = await response.json();
+            /* If DOI does not contain local prefix, try to resolve with the general handler */
+            if (doi.includes('TEST') || doi.includes('SANDBOX')) {
+                let environment: string = 'dev';
 
-            if (record.data) {
-                setActive(true);
+                if (doi.includes('SANDBOX')) {
+                    environment = 'sandbox';
```

Comment:
this is an awkward way to assign the environment var. Can you do something like: 
`let environment: doi.includes('TEST') ? string = 'dev' : string = 'sandbox'`

I think javascript uses the [conditional operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) the same way as in java

Reply:
True, in principal it does not affect any of the code game play, but in this case the one liner looks cleaner, so changed it